### DISCUSSION TITLE
Need help making a workload published on one tenant available in another tenant

### DISCUSSION THREAD

**Discussion by box-turtle** [microsoft/Microsoft-Fabric-workload-development-sample#65] [Help] [answered]
We have published our workload to our prod environment. For internal testing purposes we want to make that workload available to our corp tenant and a dev tenant.

We entered the tenant id for the two tenants and it seems like we were successful in publishing:

<img width="1011" alt="image" src="https://github.com/user-attachments/assets/9c930870-f621-46b9-8d2a-9aa5b8fd9985">

But we don't see the workload appear anywhere on the two target tenants. Is there a step that has to be performed on the tenants to add the workload? (it doesn't appear under either "My workloads" or "More workloads" tabs.

thanks

**Comment by box-turtle** [comment]
Any update on this? We were told that a fix for publishing workloads across regions was going in this past weekend, but we don't see a change. Do we need to publish our workload again for this to take effect?

**Reply by IdanGoldfarb1989** [reply, marked as answer]
Hi, there was a general issue with Fabric that caused our fix not to be deployed as planned. We expect a delay of a few days.

Sorry for the inconvenience.
